src/test/java/com/google/maps/PlacesApiIntegrationTest.java

```diff
@@ -206,10 +207,14 @@ public void testPlaceDetailsInFrench() throws Exception {
   public void testPlaceTextSearchPermanentlyClosed() throws Exception {
     PlacesSearchResponse response = PlacesApi.textSearchQuery(context, "ABC Learning Centres in australia").await();
     assertNotNull(response);
-    PlacesSearchResult result = response.results[0];
-    assertNotNull(result);
-    assertEquals("ABC Learning Centre", result.name);
-    assertEquals(true, result.permanentlyClosed);
+    for (PlacesSearchResult result: response.results) {
+      assertNotNull(result);
+      if(result.permanentlyClosed) {
+        // test success condition
```

Comment:
I've had to water the test down from asserting the first returned result for the query is permanently closed, to at least one of the returned result set is permanently closed. Because data.